pyproject.toml

```diff
@@ -12,23 +12,21 @@ aiocontextvars = "^0.2.2"
 aiohttp = "^3.10.3"
 asyncpg = "^0.29.0"
 coloredlogs = "^15.0.1"
-csv-detective = "0.7.3"
+csv-detective = "0.7.5.dev1113"
 dateparser = "^1.1.7"
 humanfriendly = "^10.0"
 marshmallow = "^3.14.1"
 minicli = "^0.5.3"
 minio = "7.2.7"
-pyarrow = "16.1.0"
-python-dateutil = "^2.8.2"
+pyarrow = "18.1.0"
+pandas = "2.2.0"
```

Comment:
I would rely on pandas installed by csv-detective, so we avoid future potential pandas versions conflicts (which are actually very common and annoying with pandas/numpy). I don't know which is the best practice but it would save us somes future headaches for sure. And since the features using pandas in the code are very related to csv-detective output, it also makes sense.

Reply:
Makes sense, I'll change that (also the requirements of `csv-detective` are rather bad, I'll make a PR to improve them)